e2e/tests/file-versions.spec.ts

```
@@ -0,0 +1,75 @@
+  }
+  return fileId
+}
+
+test.describe('File versions', () => {
+  test('deletes a version from the history modal', async ({
+    alicePage,
+    aliceDrive
+  }) => {
+    const name = `versioned-${stamp()}.txt`
+    const fileId = await createFileWithVersions(name, 2)
+    expect(await countFileVersions({ instance: ALICE.instance, fileId })).toBe(2)
```

ESLint

[error] Replace `2` with `⏎······2⏎····`

(prettier/prettier)